lib/perl/Genome/Disk/Detail/Allocation/Archiver.pm

```diff
@@ -58,64 +52,20 @@ sub archive {
-                and $allocation_object->kilobytes_requested < 1048576) {
-            confess(sprintf("Total size of files at path %s is only %s, "
-                    . "which is not greater than 1GB!",
-                    $current_allocation_path,
-                    $allocation_object->kilobytes_requested));
-        }
 
-        my $mkdir_cmd = "mkdir -p $archive_allocation_path";
-        my $cd_cmd = "cd $current_allocation_path";
-        my $tar_cmd = "/bin/ls -A | tar --create --file $tar_path -T -";
-        my $cmd = join(' && ', $mkdir_cmd, $cd_cmd, $tar_cmd);
-
-        $tarball_created = 1;
-        # It's very possible that if no commit is on, the volumes/allocations
-        # being dealt with are test objects that don't exist out of this local
-        # UR context, so bsubbing jobs would fail.
-        if ($ENV{UR_DBI_NO_COMMIT}) {
-            Genome::Sys->shellcmd(cmd => $cmd);
+        unless ($allocation_object->volume->archive_is_mounted) {
+            $self->fatal_message('volume for %s is not mounted', $allocation_object->archive_path);
         }
-        else {
-            my ($job_id, $status);
-
-            my @signals = qw/ INT TERM /;
-            for my $signal (@signals) {
-                $SIG{$signal} = sub {
-                    print STDERR "Cleanup activated within allocation, cleaning up LSF jobs\n";
-                    eval { Genome::Sys->kill_lsf_job($job_id) } if $job_id;
-                    $allocation_object->_cleanup_archive_directory($archive_allocation_path);
-                    die "Received signal, exiting.";
-                }
-            }
-            # Entire path must be wrapped in quotes because older allocation
-            # IDs contain spaces If the command isn't wrapped in quotes, the
-            # '&&' is misinterpreted by bash (rather than being "bsub '1 && 2'
-            # it is looked at as 'bsub 1' && '2')
-            my $guard = Genome::Config::set_env('lsb_sub_additional', ''); #no docker for archives
-            ($job_id, $status) = Genome::Sys->bsub_and_wait(
-                queue => Genome::Config::get('archive_lsf_queue'),
-                job_group => '/archive',
-                log_file => "/tmp/$id",
-                cmd => "$cmd",
-            );
-
-            for my $signal (@signals) {
-                delete $SIG{$signal};
-            }
-
-            unless ($status eq 'DONE') {
-                confess sprintf("LSF job %s failed to execute %s, "
-                    . "exited with status %s", $job_id, $cmd, $status);
-            }
+
+        unless ($allocation_object->volume->is_mounted) {
+            $self->fatal_message('volume for %s is not mounted', $allocation_object->absolute_path);
         }
 
-        $allocation_object->mount_path(
-            $allocation_object->volume->archive_mount_path);
-        $allocation_object->_update_owner_for_move;
+        Genome::Sys->create_directory($archive_allocation_path);
```

Comment:
If I'm reading the Genome::Sys->create_directory code correctly, this will continue even if the directory exists in the archive.  My instinct is that if the directory already exists in archive, something unexpected has happened, and we should probably throw an informative error and exit.  Happy to be convinced otherwise, though